### DISCUSSION TITLE
I'm having issues with Diablo Soloplay.

### DISCUSSION THREAD

**Discussion by sskim79** [blizzhackers/kolbot-SoloPlay#43] [Q&A about kolbot SoloPlay]
I use the Korean version of Diablo.

Act 1 Stelling 250 seconds without hunting Andariel.

Can you tell me where the problem is?

**Comment by aim2kill** [comment]
You need english diablo version

**Reply by tyson-dev** [reply]
In your profile, try adding "-direct -txt" right after "-w -sleepy -ftj" to your parameters. Worked for german version for me.
